spaceship/lib/spaceship/portal/portal_client.rb

```diff
@@ -427,20 +427,23 @@ def delete_provisioning_profile!(profile_id, mac: false)
       parse_response(r)
     end
 
-    def repair_provisioning_profile!(profile_id, name, distribution_method, app_id, certificate_ids, device_ids, mac: false)
+    def repair_provisioning_profile!(profile_id, name, distribution_method, app_id, certificate_ids, device_ids, mac: false, sub_platform: nil)
```

Comment:
@hotsjf  I meant that I don't see a test scenario for `repair_provisioning_profile` called with `sub_platform` set to `tvOS`. You said you added a test, but I don't see it! :)